mmv1/products/firestore/Index.yaml

```diff
@@ -64,6 +64,15 @@ examples:
       # so we need a pre-created project.
```

Comment:
Can you also remove "with `type` set to `"FIRESTORE_NATIVE"`" bit above?

Reply:
Done